server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -254,15 +256,19 @@ private ContainerTag blockMoveButtons(
     // Move up button is invisible for the first block
     String moveUpInvisible =
         blockDefinition.id() == blockDefinitions.get(0).id() ? Styles.INVISIBLE : "";
-    Tag moveUp =
+    DivTag moveUp =
         div()
             .withClass(moveUpInvisible)
             .with(
                 form()
                     .withAction(moveUpFormAction)
                     .withMethod(HttpVerbs.POST)
                     .with(makeCsrfTokenInputTag(request))
-                    .with(input().isHidden().withName("direction").withValue(Direction.UP.name()))
+                    .with(
+                        input()
+                            .isHidden()
+                            .attr("name", "direction")
```

Comment:
Mind switching to `withName` / `withValue`

Same for other `.attr` references in this and other files.

Reply:
Oh crap I thought I had gotten all those. Maybe those are from the merge?